apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.StatisticsTests.cs

```diff
@@ -26,7 +26,8 @@ public sealed class StatisticsTests
             public void AllProperties()
             {
                 var db1Stats = new DatabaseStatistics(
-                    databaseName: new DatabaseName("project", "instance", "db1"),
+                    new PartitionKey(
+                    new DatabaseName("project", "instance", "db1"), null),
```

Comment:
Same here, please check that we get correct statistics if there's a database role set.